.idea/modules/android.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should not be added I think. RUBY_MODULE is maybe not available on all machines ( e.g. windows etc)